src/quill-editor.component.ts

```diff
@@ -126,11 +127,11 @@ export class QuillEditorComponent implements AfterViewInit, ControlValueAccessor
 
     // update model if text changes
     this.quillEditor.on('text-change', (delta: any, oldDelta: any, source: string) => {
-      let html = this.editorElem.children[0].innerHTML;
+      let html: Html = this.editorElem.children[0].innerHTML;
```

Comment:
You're right.
It's not necessary to create a new type, using the following definition is sufficient:
`let html: (string | null) = ...`
Thanks.